src/functionalTest/java/uk/gov/hmcts/divorce/systemupdate/event/SystemNotifyApplicantDisputeFormOverdueFT.java

```diff
@@ -41,6 +42,8 @@
         "classpath:request/casedata/ccd-callback-casedata-system-notify-applicant-dispute-form-overdue.json";
     private static final String RESPONSE = "classpath:responses/response-system-notify-applicant-dispute-form-overdue.json";
 
+    @Value("${submit_aos.dispute_offset_days}")
+    private int disputeDueDateOffsetDays;
```

Comment:
why does this need to be configurable?

Reply:
because it is a magic number, I thought it might be worth making it configurable in case policy/business decide to change it